packages/manager/modules/pci/src/projects/project/public-gateways/onboarding/onboarding.controller.js

```diff
@@ -19,20 +21,37 @@ export default class PciPublicGatewaysOnboardingController {
           title: this.$translate.instant(
             `pci_projects_project_public_gateways_onboarding_guides_${guide.id}_title`,
           ),
-          description: this.$translate.instant(
-            `pci_projects_project_public_gateways_onboarding_guides_${guide.id}_description`,
-          ),
+          link: guide.links[this.user.ovhSubsidiary] || guide.links.DEFAULT,
         },
       ],
       [],
     );
   }
 
   onAddPublicGatewayClick() {
-    this.trackPublicGateways('onboarding::add');
+    this.trackPublicGateways('public-gateway::onboarding::add');
```

Comment:
'public-gateway::' is present in all tracking. Is there any way to put it in a constant as a subprefix

Reply:
Earlier I used `public-gateway` as the prefix but when I see list page has `Public-gateway` as tracking so I moved it from common prefix.